morf-testsupport/src/main/java/org/alfasoftware/morf/testing/DatabaseSchemaManager.java

```diff
@@ -358,13 +368,14 @@ private void executeScript(Collection<String> script) {
    */
   public void dropAllViews() {
     ProducerCache producerCache = new ProducerCache();
-    log.debug("Dropping all views");
+    if (log.isDebugEnabled()) log.debug("Dropping all views");
     try {
       Schema databaseSchema = producerCache.get().getSchema();
       ImmutableList<View> viewsToDrop = ImmutableList.copyOf(databaseSchema.views());
       List<String> script = Lists.newArrayList();
       for (View view : viewsToDrop) {
         script.addAll(dialect.get().dropStatements(view));
+        if (log.isTraceEnabled()) log.trace("dropAllViews: " + view.getName());

```

Comment:
same here?

Reply:
Yes, views are usually a lot less interesting in most situations :)